src/user_config.hpp

```diff
@@ -22,6 +22,9 @@ class UserConfigWrapper : public ConfigBaseImpl, public Napi::ObjectWrap<UserCon
 
     Napi::Value getEnableBlindedMsgRequest(const Napi::CallbackInfo& info);
     void setEnableBlindedMsgRequest(const Napi::CallbackInfo& info);
+
+    Napi::Value getExpiry(const Napi::CallbackInfo& info);
+    void setExpiry(const Napi::CallbackInfo& info);
```

Comment:
These are a bit confusingly named, since this is specifically setting/returning the note-to-self expiry rather than a more general User Config expiry that one might guess from the name.

Reply:
My understanding is still beginner level so forgive any incorrect assumptions I make. But I had a look through the code and couldn't spot what you meant by "general User Config expiry" the only expiry methods in User Profile are the above functions. Which are exclusive to Note to Self since that is what the User Profile is? The rest is handled in Contacts. 

Or are you talking about making the function name more explicit like "setNoteToSelfExpiry"? Audric and I chatted about this yesterday and are trying to keep the wrapper method signatures as close to the original libsession functions as possible. We will only call these functions in relation to the UserProfile so we know it is Note to Self through that.

Example session-desktop call: `userProfileWrapper.setExpiry(expirySeconds);`